### PR TITLE
docs: typography demo use react hooks

### PR DESCRIPTION
**Pull request by xiaohuoni**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/20093

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/demo/ellipsis-debug.md](https://github.com/ant-design/ant-design/blob/4.0-update-typography-demo/components/typography/demo/ellipsis-debug.md)
[View rendered components/typography/demo/interactive.md](https://github.com/ant-design/ant-design/blob/4.0-update-typography-demo/components/typography/demo/interactive.md)

### CODE REVIEW

File: components/typography/demo/ellipsis-debug.md [hunk 1]
```diff
@@ -16,57 +16,45 @@ Multiple line ellipsis support.
 
 ```jsx
 import { Typography, Slider, Switch } from 'antd';
+import { useState } from 'react';
```
Comment (afc163) [inline components/typography/demo/ellipsis-debug.md]:
```suggestion
import React, { useState } from 'react';
```

Comment (afc163) [reply components/typography/demo/ellipsis-debug.md]: 避免用户拷贝之后没写 React 报错。

File: components/typography/demo/interactive.md [hunk 2]
```diff
@@ -15,29 +15,23 @@ Provide additional interactive capacity.
...
-      <div>
-        <Paragraph editable={{ onChange: this.onChange }}>{this.state.str}</Paragraph>
-        <Paragraph copyable>This is a copyable text.</Paragraph>
-        <Paragraph copyable={{ text: 'Hello, Ant Design!' }}>Replace copy text.</Paragraph>
-      </div>
-    );
-  }
-}
+  return (
+    <div>
```
Comment (afc163) [inline components/typography/demo/interactive.md]: 可以用 `<>`

File: components/typography/demo/ellipsis-debug.md [hunk 3]
```diff
@@ -16,57 +16,45 @@ Multiple line ellipsis support.
...
-}
+const Demo = () => {
+  const [rows, setRows] = useState(1);
+  const [longText, setLongText] = useState(true);
+  const [copyable, setCopyable] = useState(false);
+  const [editable, setDditable] = useState(false);
+  const [expandable, setExpandable] = useState(false);
+
+  return (
+    <div>
```
Comment (afc163) [inline components/typography/demo/ellipsis-debug.md]: 可以用 `<>`。

**Comment by xiaohuoni** [comment]
先关闭了，后面统一重新提

**Pull request closed without merging**